src/dataframes.jl

```diff
@@ -21,12 +21,31 @@ function Plots.extractGroupArgs(group::Symbol, df::AbstractDataFrame, args...)
     Plots.extractGroupArgs(collect(df[group]))
 end
 
+# allows the passing of expressions including DataFrame columns as symbols
+function processExpr(expr, df)
+    arg = map(expr.args) do x
+        isa(x, Expr) && return processExpr(x, df)
+        if isa(x, QuoteNode) && isa(x.value, Symbol)
+            return collect(df[x.value])
```

Comment:
I don't think this is right... you don't want to collect here... you want to return a **collect expression** so that it will be called during the `eval`.  (it may work, but that requires building a giant constant expression to be evaluated)